src/components/shared/avatar-upload.tsx

```
@@ -0,0 +1,95 @@
+      setIsUploading(true);
+
+      const res = await fetch("/api/upload-photo", {
+        method: "PUT",
+        body: formData,
+      });
+
+      const data = await res.json();
+
+      // ✅ Check JSON response
+      if (!res.ok || data.status !== "success") {
+        console.error("Upload failed", data);
```

ESLint

[error] Unexpected console statement.

(no-console)